src/main/steps/common/common.content.ts

```diff
@@ -41,6 +41,7 @@ const en = {
   wife: 'wife',
   partner: 'partner',
   civilPartner: 'civil partner',
+  checkYourAnswersPartner: 'check your answers partner',
```

Comment:
shouldnt this be check your partner's answer? kinda sounds like a cowboy, howdy partner

Reply:
haha agreed it sounds cowboy-ish but I think makes more sense because it's the partner for the CYA page. 
I'll change it to checkTheirAnswersPartner as that might make more sense